src/check_random_progs.py

```diff
@@ -298,7 +297,7 @@ def check(idx, config):
             dump_file(CRASH_BUG_DIR, p4_file)
             if config["do_prune"]:
                 info_file = CRASH_BUG_DIR.joinpath(f"{p4_file.stem}_info.json")
-                info = validation.INFO
+                info = 1
```

Comment:
Hmm, instead can we copy over the definition from validation? I will deprecate those files soon.